src/test/java/com/github/princesslana/eriscasper/faker/DataFaker.java

```diff
@@ -47,6 +45,38 @@ public static User user() {
         .build();
   }
 
+  public static Guild guild() {
+    Snowflake id = DiscordFaker.snowflake();
+    return ImmutableGuild.builder()
+        .id(id)
+        .name("FakeGuild")
```

Comment:
Fields such of these should use fake rather than constant data as well. Check out what's available in http://dius.github.io/java-faker/apidocs/index.html